AppInspector/rules/default/general/code_metrics.json

```diff
@@ -7,14 +7,33 @@
       "Metric.Code.Class.Defined"
     ],
     "severity": "moderate",
-    "applies_to": [ "csharp", "cpp", "javascript", "python", "vb", "rust", "ruby", "groovy", "php" ],
+    "applies_to": [ "csharp", "cpp", "javascript", "python", "vb", "ruby", "groovy", "php" ],
     "patterns": [
       {
         "pattern": "class",
         "type": "regex-word",
         "scopes": [ "code" ],
         "confidence": "high",
-        "_comment": "langs defined to exclude css class reference vs class data structure which provides complexity/size insight"
+        "_comment": "data structure which provides complexity/size insight"
+      }
+    ]
+  },
+  {
+    "name": "Metric: Struct Definition",
+    "id": "AI025710",
+    "description": "Metric: Struct Defined",
+    "tags": [
+      "Metric.Code.Class.Defined"
+    ],
+    "severity": "moderate",
+    "applies_to": [ "c", "cpp", "rust" ],
```

Comment:
I think structs apply to Java and C# too (I'm sure more, but those off the top of my head.)

Reply:
"Java doesn't have structs like C or C++, but you can use Java classes and treat them like a struct." Added objective-c, c#